diff --git a/wasm/molt_runtime.wasm.sha256 b/wasm/molt_runtime.wasm.sha256
index 06a74cbad..73c40feb6 100644
--- a/wasm/molt_runtime.wasm.sha256
+++ b/wasm/molt_runtime.wasm.sha256
@@ -1 +1 @@
-864cca3107ac7f35f3c1e117b24533315057e99712132430cbe986fa883aa579
+6f7d1cec565b30ff45be94930425e917ccf8540b016cd309fe664f88f9cca131
diff --git a/wasm/molt_runtime_reloc.wasm.sha256 b/wasm/molt_runtime_reloc.wasm.sha256
index 0fda77466..f15260fef 100644
--- a/wasm/molt_runtime_reloc.wasm.sha256
+++ b/wasm/molt_runtime_reloc.wasm.sha256
@@ -1 +1 @@
-a040f0d6ae7f10cc1eca4a01e888c0b59e00e51a150514958b3f2261d14339f3
+54959dcc75bbb59d1a74d87164fd39a4a3be047f5bb2a9b6f14cdf301b7d6bc8
